web/hooks/use-likes.ts

```diff
@@ -21,6 +23,23 @@ export const useLikesOnContent = (
         .eq('content_type', contentType)
         .eq('content_id', contentId)
     ).then(({ data }) => setLikes(data))
+    
+    const handleNewLike = (msg: ServerMessage<'broadcast'>) => {
+      if (msg.data && (msg.data as any).comment_id === contentId) {
+        setLikes((msg.data as any).likes)
+      }
+    }
+    
+    const subscription = useApiSubscription({
+      topics: [`contract/${contentId}/new-like`],
+      onBroadcast: handleNewLike,
+    })
+    
+    return () => {
+      if (subscription && 'unsubscribe' in subscription && typeof subscription.unsubscribe === 'function') {
+        subscription.unsubscribe()
+      }
+    }
```

Comment:
The useApiSubscription does not return anything. You don't need this. It cleans up itself.